ui/src/components/friends/incoming_requests/mod.rs

```diff
@@ -83,8 +83,8 @@ pub fn PendingFriends(cx: Scope) -> Element {
         }
     });
 
-    cx.render(rsx! (
-        div {
+    cx.render(rsx!(if !friends_list.is_empty() {
```

Comment:
for this file and outgoing_requests, it would remove a level of nesting if this was done instead:

```
if friends_list.is_empty() { return render!(); }
cx.render(...)
```